packages/extension-chakra-storefront/src/theme/components/project/search.js

```diff
@@ -19,9 +19,8 @@ export default defineRecipe({
             borderColor: 'gray.600',
             border: '2px solid',
             backgroundColor: 'gray.100',
-            _focusVisible: {
-                background: 'transparent',
-                borderColor: 'blue.500'
+            _placeholder: {
```

Comment:
This style is specific to search input. I don't think it should apply to all input, I moved it here